### PR TITLE
[xrayexporter]drop invalid xray trace id

### PR DESCRIPTION
**Pull request by wyTrivail**
**Description:** <Describe what has changed.>

Drop the invalid xray trace id to reduce confusion.

**Link to tracking Issue:** <Issue number if applicable>

https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1255

**Testing:** <Describe what testing was performed and which tests were added.>

unit test passed, let me know if we need a manual e2e test to verify it.

### CODE REVIEW

File: exporter/awsxrayexporter/translator/segment.go [hunk 1]
```diff
@@ -274,10 +283,9 @@ func convertToAmazonTraceID(traceID pdata.TraceID) string {
 	// the traceID, then the epoch may be outside the accepted AWS range of within the
 	// past 30 days.
 	//
-	// In that case, we use the current time as the epoch and accept that a new span
-	// may be created
+	// In that case, we return invalid traceid error
 	if delta := epochNow - epoch; delta > maxAge || delta < -maxSkew {
-		epoch = epochNow
+		return "", errors.New("invalid xray traceid: " + traceID.HexString())
```
Comment (anuraaga) [inline exporter/awsxrayexporter/translator/segment.go:288]: Think fmt.Errorf is more conventional for creating an error from a formatted string.

**Review by anuraaga — APPROVED** [review]
Just small nit, thanks

File: exporter/awsxrayexporter/translator/segment.go (hunk 1 above)
Comment (wyTrivail) [reply exporter/awsxrayexporter/translator/segment.go:288]: Thx for pointing out! Let me make the change

**Comment by tigrannajaryan** [comment]
Does this resolve https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1255 ?

**Comment by anuraaga** [comment]
Yup sorry for not marking at Fixes

**Pull request merged**